modules/user/client-react/index.tsx

```diff
@@ -55,14 +67,14 @@ interface Errors {
 export interface FormProps<V> {
   handleSubmit: (values: V, props: HandleSubmitProps<V>) => void;
   onSubmit: (values: V) => Promise<void> | void | any;
-  submitting: boolean;
+  submitting?: boolean;
   errors: Errors;
   values: V;
   t: TranslateFunction;
 }
 
 const ProfileName: React.FunctionComponent<WithLogoutProps> = withLoadedUser(({ currentUser }) => (
-  <React.Fragment>{currentUser ? currentUser.fullName || currentUser.username : null}</React.Fragment>
+  <React.Fragment>{currentUser ? currentUser.profile.fullName || currentUser.username : null}</React.Fragment>
```

Comment:
@Iwasherd Why did you use <React.Fragment> instead of just <>?

Reply:
Done.